Projects/Team2/ForeignPassport.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Passport
+{
+	class ForeignPassport : Passport
+	{
+		public string visaDetails { get; set; }
+
+		public int seriadlnumber { get; set; }
+		
+		public ForeignPassport(string name, string lastName, int age, string sex, string placeOfResidence, string visaDetails, int seriadlnumber) :base (name, lastName, age, sex, placeOfResidence)
+		{
+			this.visaDetails = visaDetails;
+			this.seriadlnumber = seriadlnumber;
+			Console.WriteLine($"Name: {name}\n Last name: {lastName}\n Age: {age}\n Sex: {sex}\n Place of residence: {placeOfResidence}\n " +
+				$"\nVisa details: {visaDetails}\n Serial number: {seriadlnumber}");
+			Console.ReadKey();
+		}
+
```

Comment:
you should use parameters from the constructor to initialize an object, not just to print it out